src/requirements.yml

```diff
@@ -1,6 +1,10 @@
 ---
 - src: https://github.com/cisagov/ansible-role-amazon-ssm-agent
   name: amazon_ssm_agent
+- src: https://github.com/cisagov/ansible-role-automated-security-updates
+  name: automated_security_updates
+- src: https://github.com/cisagov/ansible-role-backports
+  name: backports
```

Comment:
Looks like this got re-added after being removed in https://github.com/cisagov/nessus-packer/commit/a0daf19d9f81042ab1e6bd624be4a89cacaa1cc4.

```suggestion
```